lib/auth/oauth2_authenticator.rb

```diff
@@ -1,59 +0,0 @@
-# frozen_string_literal: true
-
-class Auth::OAuth2Authenticator < Auth::Authenticator
```

Comment:
Deprecated

Reply:
What do you think about splitting the removal of depredations into separate PRs and into their own individual commits? This would make it easier for us to revert the removal should something go wrong.

Reply:
While I agree with you, the problem is that some stuff is breaking if I don’t remove it (like the deprecated sassc patches), some other will have to be renamed/changed (and it’s a bit of a shame since we’re going to remove all that) and finally some of this stuff will create more complex code in some of our initializers for the time being. So not sure about the best course of action here :thinking: